docker/ckan/2.9.9-base.Dockerfile

```diff
@@ -1,16 +1,10 @@
 FROM ghcr.io/alphagov/ckan:2.9.9-core
 
-# copy source files and copy production.ini & setup_ckan.sh
-COPY . $CKAN_VENV/src/ckanext-datagovuk/
-RUN cp -v $CKAN_VENV/src/ckanext-datagovuk/production.ini $CKAN_CONFIG/production.ini && \
-    cp -v $CKAN_VENV/src/ckanext-datagovuk/bin/setup_ckan.sh /ckan-entrypoint.sh && \
-    chmod +x /ckan-entrypoint.sh && \
-    chown -R ckan:ckan $CKAN_HOME $CKAN_VENV $CKAN_CONFIG $CKAN_STORAGE_PATH
-
+COPY production.ini $CKAN_CONFIG/production.ini
 # Set CKAN_INI
 ENV CKAN_INI $CKAN_CONFIG/production.ini
 
-WORKDIR $CKAN_VENV/src/ckanext-datagovuk/
+RUN chown -R ckan:ckan $CKAN_HOME $CKAN_VENV $CKAN_CONFIG $CKAN_STORAGE_PATH
```

Comment:
Not a problem for now, but I suspect the app doesn't really need to write to most of these. Something to revisit after launch maybe.

Reply:
Sure, I'll have a look at this after sorting out all the launch stuff, I think most of this came from the official CKAN Dockerimage